### DISCUSSION TITLE
Question about filters that has multiple condition.

### DISCUSSION THREAD

**Discussion by ghost** [erengy/taiga#1186] [Questions] [answered]
![image](https://github.com/erengy/taiga/assets/165649551/c2dc3fb5-243e-406c-a900-b1fd2a026f4c)
For example this, if I watched the ep 3 of some anime but I don't have locally will taiga download it even though I watched it?

**Comment by erengy** [comment, marked as answer]
Since the filter is set to match "Any condition", either of the conditions would suffice to trigger the action.

In this case, if the episode number of a torrent is less than or equal to your watched episodes, that torrent will be discarded (i.e. won't be downloaded). The other condition wouldn't make a difference.

In other words, you can think of "Any condition" as the `OR` operator and "All conditions" as the `AND` operator.
